packages/ckeditor5-bookmark/src/bookmarkediting.ts

```diff
@@ -44,6 +50,16 @@ export default class BookmarkEditing extends Plugin {
 			allowAttributes: 'bookmarkId',
 			disallowAttributes: 'linkHref'
 		} );
+
+		// Disallow formatting attributes on `bookmark` element.
+		schema.addAttributeCheck( ( context, attributeName ) => {
```

Comment:
Is this somehow related to update command, I feel like it more belongs to schema definition PR, but do we forget about it earlier or is it indeed a part of update command?

Reply:
Yes, and now im wondering if better would be to put it into `insert command` PR - cause tests also would suit there better. Yes, will do it this way.